packages/jellyfish-api-core/__tests__/category/loan/closeVault.test.ts

```diff
@@ -32,29 +32,20 @@ describe('Loan', () => {
   async function setup (): Promise<void> {
     // token setup
     const collateralAddress = await tGroup.get(0).container.getNewAddress()
-    await tGroup.get(0).token.dfi({ address: collateralAddress, amount: 40000 })
```

Comment:
A question here: Why do you change this file? Because the test failed right with the latest ain code ?

Reply:
nope.. i want make use of `collateralAddress` for something else